packages/fury-adapter-oas3-parser/lib/parser/oas/parseInfoObject.js

```diff
@@ -35,8 +35,17 @@ const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));
 function parseInfo(context, info) {
   const { namespace } = context;
 
+  const parseTermsOfService = (terms) => {
+    const link = new namespace.elements.Link();
+    link.relation = 'terms-of-service';
+    link.href = terms;
+
+    return link;
+  };
+
   const parseMember = R.cond([
     [hasKey('title'), parseString(context, name, true)],
+    [hasKey('termsOfService'), parseString(context, name, true)],
```

Comment:
I am not sure about the 3rd parameter, whether should be `true` or `false`

Reply:
It should be `false`, it represents that `termsOfService` is optional and not required.

Reply:
ok, change on next commit